de.metas.vertical.pharma/src/main/java/de/metas/vertical/pharma/Main.java

```diff
@@ -39,10 +43,17 @@
  */
 public class Main extends AbstractModuleInterceptor
 {
+	@Override
+	protected void onInit(IModelValidationEngine engine, I_AD_Client client)
```

Comment:
consider overriding onAfterInit() instead of onInit(). My plan is to make onInit() method final.